bbb/runner.py

```diff
@@ -23,8 +23,17 @@ def main():
     logfile = config[args.service[0]]["logfile"]
     if not logfile:
         logfile = "%s.log" % args.service
+    debug_logfile = "debug-%s" % logfile
```

Comment:
Debug log will contain all messages up to DEBUG.
On 8 Mar 2016 7:46 AM, "Ben Hearsum" <notifications@github.com> wrote:

> In bbb/runner.py
> <https://github.com/mozilla/buildbot-bridge/pull/14#discussion_r55354094>:
>
> > @@ -23,8 +23,17 @@ def main():
> >      logfile = config[args.service[0]]["logfile"]
> >      if not logfile:
> >          logfile = "%s.log" % args.service
> > +    debug_logfile = "debug-%s" % logfile
>
> Are you sure you want this logged to a separate file? It may make it
> harder to read if you need to look at debug+non-debug messages.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/buildbot-bridge/pull/14/files#r55354094>.
>